### DISCUSSION TITLE
Add myst vpn support

### DISCUSSION THREAD

**Discussion by Masterofnoothing** [engageub/InternetIncome#69] [Ideas]
Will be nice to see myst vpn support

**Comment by engageub** [comment]
Hello,
I did not understand what you mean. Could you please elaborate.
If you looking to use openvpn or multi IP with Mysterium, please use the test branch.

https://github.com/engageub/InternetIncome/tree/test

Thank you

**Reply by Masterofnoothing** [reply]
No like using mysterium vpn client to run bandwidth sharing apps

**Comment by engageub** [comment]
Yes, it can be done if the price is cheap.
Also do we have docker version for this?

**Reply by engageub** [reply]
I think it is not a good idea these days as they might already be running other bandwidth sharing apps on their own system and you may get your account blocked.

Thank you

**Reply by Masterofnoothing** [reply]
i see
